mne/utils/docs.py

```diff
@@ -3495,6 +3520,48 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 For testing the lower tail (``tail=-1``), don't subtract ``pval`` from 1.
 """
 
+docdict['time_bandwitdh_tfr'] = """
+time_bandwidth : float
+    Time x (Full) Bandwidth product. Should be ``≥ 2.0``. The frequency
+    resolution achieved depends on ``n_cycles`` and ``time_bandwdith``. For a
+    given time-window length, a finer frequency resolution is achieved by
+    decreasing ``time_bandwidth`` at the cost of the number of good tapers. See
+    notes for additional information."""
+
+docdict['time_bandwitdh_tfr_note'] = """
```

Comment:
I'll keep this one open for now. I named it this way because it mostly concerns the `time_bandwidth` argument, but it can be renamed if you prefer.

I don't know why, but I have an issue with the word `width`, one time out of 2 I spell it wrong when typing; and despite some Ctrl+F to look for those mistakes, some slipped by.